docs/release_notes.rst

```diff
@@ -25,6 +25,8 @@ New Features
 Bug Fixes
 
 * Fix ``--match`` option to only consider filename when matching full paths (#550).
+* Fix decorator parsing for async function. Resolves some false positives
```

Comment:
@cdce8p thanks for the PR! Could you please move the release notes under a 6.2.1 heading instead? Thanks!